### PR TITLE
Meeting Login personGuid and Logout

### PR DESCRIPTION
**Pull request by tkenakka**

### CODE REVIEW

**Comment by tkenakka** [comment]
Notice: this requires also and update for the config file project!

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 1]
```diff
@@ -33,24 +33,25 @@ angular.module('dashboard')
...
+        var mtgPersonGuid = StorageSrv.getKey(CONST.KEY.MEETING_PERSONGUID);
         var mtgItem = StorageSrv.getKey(CONST.KEY.MEETING_ITEM);
 
         // FUNTIONS
 
-        self.logOut = function logOutFn(meetingItem, meetingRole) {
-            $log.debug("meetingStatusCtrl.logOut");
-            if (angular.isObject(meetingItem) && angular.isObject(meetingRole)) {
+        self.logOut = function logOutFn() {
+            $log.debug("meetingStatusCtrl.logOut: \n - meeting:\n" + JSON.stringify(mtgItem) + "\n - role: " + JSON.stringify(mtgRole) + "\n - mtgPersonGuid: " + mtgPersonGuid);
```
Comment (evidentti) [inline app/scripts/controllers/meeting.status.ctrl.js]: Public function are located below private functions.

File: app/scripts/controllers/overview.ctrl.js [hunk 2]
```diff
@@ -66,18 +66,19 @@ angular.module('dashboard')
 
         localStorage.overviewState = mode;
 
-        function goToMeeting(meetingItem, meetingRole) {
+        function goToMeeting(meetingItem, meetingRole, personGuid) {
```
Comment (evidentti) [inline app/scripts/controllers/overview.ctrl.js:69]: Should there be a check if parameters exists?

**Comment by evidentti** [comment]
LGTM with comments

**Pull request merged**